From 5e24c460a043a5e120095a2470273470c51ffeb2 Mon Sep 17 00:00:00 2001
From: "shoyeb.khan" <shoyeb.khan@lge.com>
Date: Wed, 27 Jan 2021 03:34:27 +0530
Subject: [PATCH] Enable mesh & fixed ell undefined symbol error

:Release Notes:
Enable mesh with Bluez 5.55 in WebOS OSE

:Detailed Notes:
Enabled mesh in Bluez 5.55 and fixed ell
undefined symbol error

:Testing Performed:
Builded and tested

:QA Notes:
NA

:Issues Addressed:
[PLAT-131628] Enable mesh with Bluez 5.55 in WebOS OSE

---
Upstream-Status: Inappropriate [webos specific]

 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 17623b103..99c55455a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -698,7 +698,7 @@ ell/internal: Makefile
 		fi \
 	done > $@
 
-ell/ell.h: Makefile
+ell/ell_h: Makefile
 	$(AM_V_at)echo -n > $@
 	$(AM_V_GEN)for f in $(ell_headers) ; do \
 		echo "#include <$$f>" >> $@ ; \
